lib/diagnostic.rb

```diff
@@ -115,10 +118,12 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = Hash.new(''){age: 23, first_name: 'Rob'}
+
+'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = 'person.default = "#{first_name} is a mensch!"'
```

Comment:
You need to identify the object name and then the key. 

`Diagnostic.hash_default = 'person.default = "#person[:first_name] is a mensch!"`